src/python_testing/TC_MCORE_FS_1_1.py

```diff
@@ -47,7 +47,7 @@
 
 import chip.clusters as Clusters
 from chip import ChipDeviceCtrl
-from chip.testing.tasks import Subprocess
+from matter_testing_infrastructure.chip.testing.tasks import Subprocess
```

Comment:
```suggestion
try:
    from chip.testing.tasks import Subprocess
except:
    from matter_testing_infrastructure.chip.testing.tasks import Subprocess
```

Reply:
These are workarounds of not having the chip test wheel installed. Test harness should install the wheel.